iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Identifiable.java

```diff
@@ -35,6 +37,34 @@
 
     /**
      * Get properties associated to the object.
+     *
+     * @deprecated Use {@link #getProperty()} & {@link #setProperty()} instead.
      */
+    @Deprecated
     Properties getProperties();
```

Comment:
Add a default implementation that throws an exception (UnsupportedOperationException).